pkg/capabilities/capabilities.go

```diff
@@ -112,35 +115,42 @@ func MergeCapabilities(base, adds, drops []string) ([]string, error) {
 		return nil, err
 	}
 
-	// Make sure that capDrop and capAdd are distinct sets.
+	if stringInSlice(All, capDrop) {
+		// "Drop" all capabilities; return what's in capAdd instead
+		return capAdd, nil
+	}
+
+	if stringInSlice(All, capAdd) {
```

Comment:
Do we want to support `--cap-add all --cap-drop CAP_SYS_ADMIN`?

Reply:
That can be handled by passing in all on the base, Nothing on the capadd and dropining SYS_ADMIN.